src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -527,6 +511,21 @@ export class DBTCoreProjectIntegration
     return macroPaths;
   }
 
+  private async findProfilesDirectory(): Promise<string> {
+    const profilesDir = await this.python.lock(
+      (python) =>
+        python`default_profiles_dir(Path(${this.projectRoot.fsPath}))`,
+    );
+    return profilesDir;
+  }
+
+  private async getTargetDirectory(): Promise<string> {
+    const targetDir = await this.python.lock(
+      (python) => python`target_path(${this.projectRoot.fsPath})`,
```

Comment:
This line before was `python`target_path(Path(${this.projectRoot.fsPath}))`,`, That's the only place where we were using a pathlib PosixPath. It was not necessary and for some reason it didn't throw any errors in local testing.